UI_UX/Frontend/src/components/ExchangeRate.js

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import { Loading } from 'carbon-components-react';
+
+//value = -1 => loading state
+export const ExchangeRate = ({ value }) => {
+  let loadingIcon = undefined;
+  if (value == -1)
+    loadingIcon = (
+      <Loading
+        className="AppHeader_exchange_rate_loading"
+        small
+        withOverlay={false}
+      />
+    );
+  return (
+    <div>
+      {loadingIcon}
+      <span className="AppHeader_exchange_rate">
+        {value == -1 ? `` : `$${value}`} <span className="bold">USD/MYB</span>

```

Comment:
We need to start using constants for hardcoded values. So in trading lingo, this is a symbol, so we could use `const USD_MYB_SYMBOL = 'USD/MYB'. Then import it from `constants/index.js` as `import { USD_MYB_SYMBOL } from './constants'` and use it, so we don't need to repeat it or change it when we do `EUR/MYB` for instance.